CRITICAL - 2022-12-16 13:18:19 --> Invalid file: venta/registrar/index.php
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('venta/registrar/index.php')
 2 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('venta/registrar/index', [], true)
 3 APPPATH\Controllers\RegistrarController.php(28): view('venta/registrar/index')
 4 SYSTEMPATH\CodeIgniter.php(898): App\Controllers\RegistrarController->index()
 5 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegistrarController))
 6 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(43): require_once('FCPATH\\index.php')
ERROR - 2022-12-16 20:11:02 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`delivery`.`pedidos`, CONSTRAINT `pedidos_empresa_ibfk_` FOREIGN KEY (`idempresa`) REFERENCES `empresas` (`idempresa`) ON DELETE CASCADE ON UPDATE CASCADE) in /home/merino/Documentos/desarrollo/delivery/system/Database/MySQLi/Connection.php:292
Stack trace:
#0 /home/merino/Documentos/desarrollo/delivery/system/Database/MySQLi/Connection.php(292): mysqli->query()
#1 /home/merino/Documentos/desarrollo/delivery/system/Database/BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute()
#2 /home/merino/Documentos/desarrollo/delivery/system/Database/BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 /home/merino/Documentos/desarrollo/delivery/system/Database/BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query()
#4 /home/merino/Documentos/desarrollo/delivery/system/Model.php(278): CodeIgniter\Database\BaseBuilder->insert()
#5 /home/merino/Documentos/desarrollo/delivery/system/BaseModel.php(738): CodeIgniter\Model->doInsert()
#6 /home/merino/Documentos/desarrollo/delivery/system/Model.php(666): CodeIgniter\BaseModel->insert()
#7 /home/merino/Documentos/desarrollo/delivery/app/Controllers/PedidoController.php(176): CodeIgniter\Model->insert()
#8 /home/merino/Documentos/desarrollo/delivery/system/CodeIgniter.php(898): App\Controllers\PedidoController->createPedido()
#9 /home/merino/Documentos/desarrollo/delivery/system/CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController()
#10 /home/merino/Documentos/desarrollo/delivery/system/CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest()
#11 /home/merino/Documentos/desarrollo/delivery/public/index.php(67): CodeIgniter\CodeIgniter->run()
#12 /home/merino/Documentos/desarrollo/delivery/system/Commands/Server/rewrite.php(43): require_once('...')
#13 {main}
ERROR - 2022-12-16 20:12:00 --> mysqli_sql_exception: Column 'idempresa' cannot be null in /home/merino/Documentos/desarrollo/delivery/system/Database/MySQLi/Connection.php:292
Stack trace:
#0 /home/merino/Documentos/desarrollo/delivery/system/Database/MySQLi/Connection.php(292): mysqli->query()
#1 /home/merino/Documentos/desarrollo/delivery/system/Database/BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute()
#2 /home/merino/Documentos/desarrollo/delivery/system/Database/BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 /home/merino/Documentos/desarrollo/delivery/system/Database/BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query()
#4 /home/merino/Documentos/desarrollo/delivery/system/Model.php(278): CodeIgniter\Database\BaseBuilder->insert()
#5 /home/merino/Documentos/desarrollo/delivery/system/BaseModel.php(738): CodeIgniter\Model->doInsert()
#6 /home/merino/Documentos/desarrollo/delivery/system/Model.php(666): CodeIgniter\BaseModel->insert()
#7 /home/merino/Documentos/desarrollo/delivery/app/Controllers/PedidoController.php(176): CodeIgniter\Model->insert()
#8 /home/merino/Documentos/desarrollo/delivery/system/CodeIgniter.php(898): App\Controllers\PedidoController->createPedido()
#9 /home/merino/Documentos/desarrollo/delivery/system/CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController()
#10 /home/merino/Documentos/desarrollo/delivery/system/CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest()
#11 /home/merino/Documentos/desarrollo/delivery/public/index.php(67): CodeIgniter\CodeIgniter->run()
#12 /home/merino/Documentos/desarrollo/delivery/system/Commands/Server/rewrite.php(43): require_once('...')
#13 {main}
ERROR - 2022-12-16 20:12:58 --> mysqli_sql_exception: Column 'idempresa' cannot be null in /home/merino/Documentos/desarrollo/delivery/system/Database/MySQLi/Connection.php:292
Stack trace:
#0 /home/merino/Documentos/desarrollo/delivery/system/Database/MySQLi/Connection.php(292): mysqli->query()
#1 /home/merino/Documentos/desarrollo/delivery/system/Database/BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute()
#2 /home/merino/Documentos/desarrollo/delivery/system/Database/BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 /home/merino/Documentos/desarrollo/delivery/system/Database/BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query()
#4 /home/merino/Documentos/desarrollo/delivery/system/Model.php(278): CodeIgniter\Database\BaseBuilder->insert()
#5 /home/merino/Documentos/desarrollo/delivery/system/BaseModel.php(738): CodeIgniter\Model->doInsert()
#6 /home/merino/Documentos/desarrollo/delivery/system/Model.php(666): CodeIgniter\BaseModel->insert()
#7 /home/merino/Documentos/desarrollo/delivery/app/Controllers/PedidoController.php(176): CodeIgniter\Model->insert()
#8 /home/merino/Documentos/desarrollo/delivery/system/CodeIgniter.php(898): App\Controllers\PedidoController->createPedido()
#9 /home/merino/Documentos/desarrollo/delivery/system/CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController()
#10 /home/merino/Documentos/desarrollo/delivery/system/CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest()
#11 /home/merino/Documentos/desarrollo/delivery/public/index.php(67): CodeIgniter\CodeIgniter->run()
#12 /home/merino/Documentos/desarrollo/delivery/system/Commands/Server/rewrite.php(43): require_once('...')
#13 {main}
ERROR - 2022-12-16 20:14:27 --> mysqli_sql_exception: Column 'idempresa' cannot be null in /home/merino/Documentos/desarrollo/delivery/system/Database/MySQLi/Connection.php:292
Stack trace:
#0 /home/merino/Documentos/desarrollo/delivery/system/Database/MySQLi/Connection.php(292): mysqli->query()
#1 /home/merino/Documentos/desarrollo/delivery/system/Database/BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute()
#2 /home/merino/Documentos/desarrollo/delivery/system/Database/BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 /home/merino/Documentos/desarrollo/delivery/system/Database/BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query()
#4 /home/merino/Documentos/desarrollo/delivery/system/Model.php(278): CodeIgniter\Database\BaseBuilder->insert()
#5 /home/merino/Documentos/desarrollo/delivery/system/BaseModel.php(738): CodeIgniter\Model->doInsert()
#6 /home/merino/Documentos/desarrollo/delivery/system/Model.php(666): CodeIgniter\BaseModel->insert()
#7 /home/merino/Documentos/desarrollo/delivery/app/Controllers/PedidoController.php(176): CodeIgniter\Model->insert()
#8 /home/merino/Documentos/desarrollo/delivery/system/CodeIgniter.php(898): App\Controllers\PedidoController->createPedido()
#9 /home/merino/Documentos/desarrollo/delivery/system/CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController()
#10 /home/merino/Documentos/desarrollo/delivery/system/CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest()
#11 /home/merino/Documentos/desarrollo/delivery/public/index.php(67): CodeIgniter\CodeIgniter->run()
#12 /home/merino/Documentos/desarrollo/delivery/system/Commands/Server/rewrite.php(43): require_once('...')
#13 {main}
ERROR - 2022-12-16 20:55:35 --> mysqli_sql_exception: Unknown column 'pedidos.idpedidoo' in 'where clause' in /home/merino/Documentos/desarrollo/delivery/system/Database/MySQLi/Connection.php:292
Stack trace:
#0 /home/merino/Documentos/desarrollo/delivery/system/Database/MySQLi/Connection.php(292): mysqli->query()
#1 /home/merino/Documentos/desarrollo/delivery/system/Database/BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute()
#2 /home/merino/Documentos/desarrollo/delivery/system/Database/BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 /home/merino/Documentos/desarrollo/delivery/system/Database/BaseBuilder.php(2078): CodeIgniter\Database\BaseConnection->query()
#4 /home/merino/Documentos/desarrollo/delivery/system/Model.php(337): CodeIgniter\Database\BaseBuilder->update()
#5 /home/merino/Documentos/desarrollo/delivery/system/BaseModel.php(865): CodeIgniter\Model->doUpdate()
#6 /home/merino/Documentos/desarrollo/delivery/system/Model.php(692): CodeIgniter\BaseModel->update()
#7 /home/merino/Documentos/desarrollo/delivery/app/Controllers/PedidoDetalleController.php(37): CodeIgniter\Model->update()
#8 /home/merino/Documentos/desarrollo/delivery/system/CodeIgniter.php(898): App\Controllers\PedidoDetalleController->create()
#9 /home/merino/Documentos/desarrollo/delivery/system/CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController()
#10 /home/merino/Documentos/desarrollo/delivery/system/CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest()
#11 /home/merino/Documentos/desarrollo/delivery/public/index.php(67): CodeIgniter\CodeIgniter->run()
#12 /home/merino/Documentos/desarrollo/delivery/system/Commands/Server/rewrite.php(43): require_once('...')
#13 {main}
ERROR - 2022-12-16 21:23:02 --> mysqli_sql_exception: Unknown table 'delivery.pedido_detalle' in /home/merino/Documentos/desarrollo/delivery/system/Database/MySQLi/Connection.php:292
Stack trace:
#0 /home/merino/Documentos/desarrollo/delivery/system/Database/MySQLi/Connection.php(292): mysqli->query()
#1 /home/merino/Documentos/desarrollo/delivery/system/Database/BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute()
#2 /home/merino/Documentos/desarrollo/delivery/system/Database/BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 /home/merino/Documentos/desarrollo/delivery/system/Database/BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query()
#4 /home/merino/Documentos/desarrollo/delivery/system/Model.php(222): CodeIgniter\Database\BaseBuilder->get()
#5 /home/merino/Documentos/desarrollo/delivery/system/BaseModel.php(584): CodeIgniter\Model->doFindAll()
#6 /home/merino/Documentos/desarrollo/delivery/app/Controllers/PedidoController.php(54): CodeIgniter\BaseModel->findAll()
#7 /home/merino/Documentos/desarrollo/delivery/system/CodeIgniter.php(898): App\Controllers\PedidoController->list()
#8 /home/merino/Documentos/desarrollo/delivery/system/CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController()
#9 /home/merino/Documentos/desarrollo/delivery/system/CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest()
#10 /home/merino/Documentos/desarrollo/delivery/public/index.php(67): CodeIgniter\CodeIgniter->run()
#11 /home/merino/Documentos/desarrollo/delivery/system/Commands/Server/rewrite.php(43): require_once('...')
#12 {main}
ERROR - 2022-12-16 21:24:35 --> mysqli_sql_exception: Unknown column 'pedido_detalle.idpedido' in 'where clause' in /home/merino/Documentos/desarrollo/delivery/system/Database/MySQLi/Connection.php:292
Stack trace:
#0 /home/merino/Documentos/desarrollo/delivery/system/Database/MySQLi/Connection.php(292): mysqli->query()
#1 /home/merino/Documentos/desarrollo/delivery/system/Database/BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute()
#2 /home/merino/Documentos/desarrollo/delivery/system/Database/BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 /home/merino/Documentos/desarrollo/delivery/system/Database/BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query()
#4 /home/merino/Documentos/desarrollo/delivery/system/Model.php(222): CodeIgniter\Database\BaseBuilder->get()
#5 /home/merino/Documentos/desarrollo/delivery/system/BaseModel.php(584): CodeIgniter\Model->doFindAll()
#6 /home/merino/Documentos/desarrollo/delivery/app/Controllers/PedidoController.php(54): CodeIgniter\BaseModel->findAll()
#7 /home/merino/Documentos/desarrollo/delivery/system/CodeIgniter.php(898): App\Controllers\PedidoController->list()
#8 /home/merino/Documentos/desarrollo/delivery/system/CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController()
#9 /home/merino/Documentos/desarrollo/delivery/system/CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest()
#10 /home/merino/Documentos/desarrollo/delivery/public/index.php(67): CodeIgniter\CodeIgniter->run()
#11 /home/merino/Documentos/desarrollo/delivery/system/Commands/Server/rewrite.php(43): require_once('...')
#12 {main}
ERROR - 2022-12-16 21:24:55 --> mysqli_sql_exception: Unknown column 'pedido_detalle.idmenudetalle' in 'on clause' in /home/merino/Documentos/desarrollo/delivery/system/Database/MySQLi/Connection.php:292
Stack trace:
#0 /home/merino/Documentos/desarrollo/delivery/system/Database/MySQLi/Connection.php(292): mysqli->query()
#1 /home/merino/Documentos/desarrollo/delivery/system/Database/BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute()
#2 /home/merino/Documentos/desarrollo/delivery/system/Database/BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 /home/merino/Documentos/desarrollo/delivery/system/Database/BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query()
#4 /home/merino/Documentos/desarrollo/delivery/system/Model.php(222): CodeIgniter\Database\BaseBuilder->get()
#5 /home/merino/Documentos/desarrollo/delivery/system/BaseModel.php(584): CodeIgniter\Model->doFindAll()
#6 /home/merino/Documentos/desarrollo/delivery/app/Controllers/PedidoController.php(54): CodeIgniter\BaseModel->findAll()
#7 /home/merino/Documentos/desarrollo/delivery/system/CodeIgniter.php(898): App\Controllers\PedidoController->list()
#8 /home/merino/Documentos/desarrollo/delivery/system/CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController()
#9 /home/merino/Documentos/desarrollo/delivery/system/CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest()
#10 /home/merino/Documentos/desarrollo/delivery/public/index.php(67): CodeIgniter\CodeIgniter->run()
#11 /home/merino/Documentos/desarrollo/delivery/system/Commands/Server/rewrite.php(43): require_once('...')
#12 {main}
ERROR - 2022-12-16 21:24:57 --> mysqli_sql_exception: Unknown column 'pedido_detalle.idmenudetalle' in 'on clause' in /home/merino/Documentos/desarrollo/delivery/system/Database/MySQLi/Connection.php:292
Stack trace:
#0 /home/merino/Documentos/desarrollo/delivery/system/Database/MySQLi/Connection.php(292): mysqli->query()
#1 /home/merino/Documentos/desarrollo/delivery/system/Database/BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute()
#2 /home/merino/Documentos/desarrollo/delivery/system/Database/BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 /home/merino/Documentos/desarrollo/delivery/system/Database/BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query()
#4 /home/merino/Documentos/desarrollo/delivery/system/Model.php(222): CodeIgniter\Database\BaseBuilder->get()
#5 /home/merino/Documentos/desarrollo/delivery/system/BaseModel.php(584): CodeIgniter\Model->doFindAll()
#6 /home/merino/Documentos/desarrollo/delivery/app/Controllers/PedidoController.php(54): CodeIgniter\BaseModel->findAll()
#7 /home/merino/Documentos/desarrollo/delivery/system/CodeIgniter.php(898): App\Controllers\PedidoController->list()
#8 /home/merino/Documentos/desarrollo/delivery/system/CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController()
#9 /home/merino/Documentos/desarrollo/delivery/system/CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest()
#10 /home/merino/Documentos/desarrollo/delivery/public/index.php(67): CodeIgniter\CodeIgniter->run()
#11 /home/merino/Documentos/desarrollo/delivery/system/Commands/Server/rewrite.php(43): require_once('...')
#12 {main}
